README.md

```diff
@@ -5,172 +5,46 @@
 [![Packagist Version](https://img.shields.io/packagist/v/datadog/php-datadogstatsd.svg?style=flat-square)](https://packagist.org/packages/datadog/php-datadogstatsd)
 [![Total Downloads](https://img.shields.io/packagist/dt/datadog/php-datadogstatsd.svg?style=flat-square)](https://packagist.org/packages/datadog/php-datadogstatsd)
 
+This is an extremely simple PHP [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/?tab=php) client.
 
-This is an extremely simple PHP [datadogstatsd](http://www.datadoghq.com/) client.
-Requires PHP >= 5.6.0.
+**Requires PHP >= 5.6.0.**
 
 See [CHANGELOG.md](CHANGELOG.md) for changes.
 
 *For a Laravel-specific implementation that wraps this library, check out [laravel-datadog-helper](https://github.com/chaseconey/laravel-datadog-helper).*
 
 ## Installation
 
-### Composer
-
 Add the following to your `composer.json`:
 
 ```
 "datadog/php-datadogstatsd": "1.4.*"
 ```
+The first version shipped in composer is *0.0.3*
 
-Note: The first version shipped in composer is 0.0.3
-
-### Or manually
+Or manually clone the repository at [github.com/DataDog/php-datadogstatsd](https://github.com/DataDog/php-datadogstatsd) and set it up with `require './src/DogStatsd.php'.`
```

Comment:
Could we just say `clone this repository and set` ? It's weird to link the repo from the README.